packages/amplify-cli/src/lib/initialize-env.js

```diff
@@ -52,6 +50,8 @@ async function initializeEnv(context) {
 
     spinner.start(`Initializing your environment: ${currentEnv}`);
     await sequential(initializationTasks);
+    spinner.succeed('Initialized your environment successfully.');
```

Comment:
Initialized provider successfully. Since the complete initialization is not done until the initEnv() of all the categories are called. Probably should add context.print.success('Initialized your environment successfully.') after category initialization is done.